.github/workflows/ci.yaml

```
@@ -1,12 +1,12 @@
-name: Tests and Linters 🧪
+name: CI 🧪
```

Just to keep with us called the intergration one `Integration Tests: 🧪`
```suggestion
name: Linters 🖌️
```

---

I quite like CI, I took inspiration from [ruff](https://github.com/astral-sh/ruff/blob/main/.github/workflows/ci.yaml). They have all tests/builds/linting in `ci.yaml` which is what this can be if we every add unit tests for things like utils or other continuous integration things

---

Ah, makes sense. Okay, let's keep CI in that case.